pandas/io/tests/parser/test_network.py

```diff
@@ -27,51 +34,15 @@ def setUp(self):
                          'pandas/io/tests/parser/data/salaries.csv')
 
     @tm.network
-    def test_url_gz(self):
-        url = self.base_url + '.gz'
-        url_table = read_table(url, compression="gzip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_bz2(self):
-        url = self.base_url + '.bz2'
-        url_table = read_table(url, compression="bz2", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_xz(self):
-        url = self.base_url + '.xz'
-        url_table = read_table(url, compression="zip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_zip(self):
-        url = self.base_url + '.zip'
-        url_table = read_table(url, compression="zip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
+    def test_compressed_urls(self):
```

Comment:
@jorisvandenbossche, to try and reduce the repetition of the testing code, I made this commit (28373f1ecc0d3190dc18d98a86849ff9bbb0b841) which uses [nose test generators](http://nose.readthedocs.io/en/latest/writing_tests.html#test-generators). However, the nose doc states:

> Please note that method generators are not supported in unittest.TestCase subclasses.

As a result, the tests yielded by `test_compressed_urls` aren't actually running. I'm attempting to run the test using:

```sh
nosetests --verbosity=3 pandas/io/tests/parser/test_network.py:TestCompressedUrl.test_compressed_urls
```

Do you know the best way to proceed? I'm new to nose and pretty new to testing, so any advice will be appreciated.